Cleanup docs (#1591)
jupyter/docker-stacks: docs/contributing/packages.md

# Package Updates

As a general rule, we do not pin package versions in our `Dockerfile`s.
The dependencies resolution is a difficult thing to do
This means that packages might have old versions.
Images are rebuilt weekly, so usually, packages receive updates quite frequently.

> _Note: We pin major.minor version of python, so it will stay the same even after `mamba update` command._

Reviewer: ````suggestion
```{note}
We pin major.minor version of python, so it will stay the same even after `mamba update` command.
```
````

# Package Updates

As a general rule, we do not pin package versions in our `Dockerfile`s.

> The dependencies resolution is a difficult thing to do

Author: ```suggestion
The dependencies resolution is a difficult thing to do.
```

This means that packages might have old versions.
Images are rebuilt weekly, so usually, packages receive updates quite frequently.

```{note}

> We pin major.minor version of python, so it will stay the same even after `mamba update` command.

Reviewer 2: ```suggestion
We pin major.minor version of python, so this will stay the same even after invoking the `mamba update` command.
```
